src/parser/druid/feral/modules/azeritetraits/GushingLacerations.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import { calculateAzeriteEffects } from 'common/stats';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import calculateBonusAzeriteDamage from 'parser/core/calculateBonusAzeriteDamage';
+import StatTracker from 'parser/shared/modules/StatTracker';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import ItemDamageDone from 'interface/others/ItemDamageDone';
+
+import Abilities from '../Abilities';
+
+const debug = false;
+
+/**
+ * Gushing Lacerations
+ * Rip deals X additional periodic damage, and has a 6% chance to award a combo point each time it deals damage.
+ * The X additional damage is applied to every tick of the DoT. The 6% chance doesn't increase with multiple traits.
+ */
+class GushingLacerations extends Analyzer {
+  static dependencies = {
+    statTracker: StatTracker,
+    abilities: Abilities,
+  };
+
+  traitBonus = 0;
+  ripTickCoefficient = 0;
+  attackPower = 0;
+  totalDamage = 0;
+  comboProcs = 0;
+  wastedCombo = 0;
+
+  constructor(...args) {
+    super(...args);
+    if (!this.selectedCombatant.hasTrait(SPELLS.GUSHING_LACERATIONS_TRAIT.id)) {
```

Comment:
Unusual setting of the `active` flag but last time I checked, the default was indeed `true` so this is ok. Just caught my attention because usually it's 
```javascript
this.active = ...;
if (!this.active) {
   return;
}
```